apps/docs/pages/guides/cli/seeding-your-database.mdx

```diff
@@ -56,36 +56,39 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
 
 You can generate seed data for local development using [Snaplet](https://www.snaplet.dev/).
 
-Let's generate some seed data for a typical blog application, with the following schema:
+If this is your first time using Snaplet to seed your project, you'll need to setup Snaplet
+for your project with `npx snaplet setup`, and follow the interactive prompts. `snaplet setup`
+will look at your database and its structure, and use it to generate the configuration files
+and assets needed by `@snaplet/seed`.
+
+Suppose we have a database with the following schema:
 
 ![An example schema](/docs/img/guides/cli/snaplet-example-schema.png)
 
-We can use the Snaplet configuration file to define the values you want to generate. For example
+We can use the Snaplet configuration file `seed.mts` to define the values we want to generate. For example
 
 - A `Post` with the title `"There is a lot of snow around here!"`
-- The `Post.createdBy` user with an email address ending in `"@acme.org"`
+- The `Post.author` user with an email address ending in `"@acme.org"`
```

Comment:
Just looking at the schema [here](https://github.com/supabase/supabase/blob/cb70067743cd3d414fea9b5f6658b03597c1dc3a/apps/docs/public/img/guides/cli/snaplet-example-schema.png). I think its okay to keep it as `createdBy`, that way we don't have to change the image?

Reply:
Ah shoot I actually thought that and then somehow went ahead and changed them anyways; thanks!